app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class ProfilesController < Api::V1::BaseController
+      MESSAGES = {
+        '200': nil,
+        '404': 'User not found',
+        '403': 'May be out of date'
+      }.freeze
```

Comment:
this is creating a hash whose keys are numbers parsed to strings parsed to symbols, which is weird

This suggestion changes it for a hash whose keys are numbers
```suggestion
      MESSAGES = {
        200 => nil,
        404 => 'User not found',
        403 => 'May be out of date'
      }.freeze
```